pom.xml

```diff
@@ -277,6 +277,48 @@
                 <artifactId>cxf-core</artifactId>
                 <version>${cxf-bundle.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.cxf.services.sts</groupId>
+                <artifactId>cxf-services-sts-core</artifactId>
+                <version>${cxf-bundle.version}</version>
+                <exclusions>
+                    <exclusion>
```

Comment:
we don't need to add these exclusions in the paren pom. if child pom are properly excluding them, then it should be fine

Reply:
Removed the exclusions from the parent pom